src/main/java/com/mvplevel1/service/OfferService.java

```diff
@@ -0,0 +1,132 @@
+import java.util.List;
+
+public class OfferService {
+
+    private static final Logger LOGGER = Logger.getLogger(OfferService.class);
+
+    private final List<LoanOfferDTO> loanOffers = new ArrayList<>();
+    private String emailPattern = "^(.+)@(.+)$";
+    private String regex1 = "^[A-Z][a-z]{2,}(?: [A-Z][a-z]*)*$";
+
+    public List<LoanOfferDTO> offers(LoanApplicationRequestDTO requestDTO )
+    {
+
+
+        if (requestDTO.getFirstName().matches(regex1) && requestDTO.getLastName().matches(regex1))
+        {
+                if (requestDTO.getTerm() >= 6) {
+                    if (requestDTO.getEmail().matches(emailPattern)) {
+                        if ((Integer.parseInt(requestDTO.getPassportNumber()) == 6) && (Integer.parseInt(requestDTO.getPassportSeries()) == 4))
+                        {
+
+                            if ((Period.between(requestDTO.getBirthdate() , LocalDate.now()).getYears() >= 18))
+                            {
+                                loanOffers.add(new LoanOfferDTO(false, false));
+                                LOGGER.info(requestDTO.getAmount() +" , 2 years, 10%, no services");
+                                loanOffers.add(new LoanOfferDTO(true, false));
+                                LOGGER.info(requestDTO.getAmount() + " , 2 years, 8%, “bank salary client” service");
+                                loanOffers.add(new LoanOfferDTO(false, true));
+                                LOGGER.info(requestDTO.getAmount() + " , 2 years, 6%, “insurance” service (insurance price put into loan’s body)");
+                               loanOffers.add(new LoanOfferDTO(true, true));
+                                LOGGER.info(requestDTO.getAmount() +" , 2 years, 4%, “insurance” and “salary client” services (insurance price put into loan’s body)");
+                            }
+                            else {
+                                LOGGER.error("age");
+                            }
+                        }
+                        else {
+                            LOGGER.error("passport");
+                        }
+
+
+                    }
+                    else {
+
+                        LOGGER.error("email");
+
+                    }
+                }
+                else {
+                    LOGGER.error("term");
+                }
+        }
+        else {
+            LOGGER.error("Please enter a valid name");
+        }
+
+        return loanOffers;
+    }
+
+
+    public CreditDTO offerLoanCalculation(ScoringDataDTO requestDto)
+    {
+
+        BigDecimal monthlyPayment;
+        BigDecimal totalCost;
+
+        CreditDTO response = new CreditDTO();
+        BigDecimal rate = new BigDecimal(10);
+        response.setRate(rate);
+        rate = response.getRate();
+
+
+        if(requestDto.getEmployment().getEmploymentStatus().toString().matches(EmploymentStatus.UNEMPLOYED.name())){
+             LOGGER.error("Sorry you don't qualify for a loan");
```

Comment:
If this condition will pass, code will normally go further, but it doesn't have to. 
The simpliest way is to throw exceptions in such cases.
Also, if you check several conditions, you may add "deny causes" in list, and at the end check if this list is empty, and if not, throw exception with all deny causes.
It is good practise to create your own exceptions for business-purpose exception situations